qutebrowser/javascript/greasemonkey_wrapper.js

```diff
@@ -1,13 +1,13 @@
 (function() {
-    const _qute_script_id = "__gm_" + {{ scriptName | tojson }};
+    const _qute_script_id = "__gm_" + "{{ scriptName }}";
```

Comment:
It seems the `" + "` from the middle of this can be removed now that it is being inserted as string-parsable chars rather than an actual json string with quotes on the ends. If you don't mind.

Reply:
I'll fix this up while merging.